api/models/task.go

```diff
@@ -23,7 +23,7 @@ const (
 
 const (
 	// FormatDefault ...
-	FormatDefault = ""
+	FormatDefault = "default"
```

Comment:
Nothing bad will happen. And if they update their route once, their format will be updated correctly.